brd-ios/breadwallet/src/Redesign/Core/Configurations/Presets.swift

```diff
@@ -83,7 +83,7 @@ extension Presets {
                                                  selectedConfiguration: Presets.Background.Primary.selected,
                                                  disabledConfiguration: Presets.Background.Primary.disabled)
         
-        static var secondary = ButtonConfiguration(backgroundConfiguration: Presets.Background.Secondary.selected.withBorder(border: Presets.Border.normal),
+        static var secondary = ButtonConfiguration(backgroundConfiguration: Presets.Background.Secondary.selected.withBorder(border: Presets.Border.selected),
```

Comment:
this config will not affect other views' configs? just for that popup? we just want to update the color on that popup. or is it universal?

Reply:
as I saw from the designs the secondary button need to be with blue border